src/components/LargeDisplayWrapper/LargeDisplayWrapper.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'lodash';
+import useLocalize from '../../hooks/useLocalize';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import BlockingView from '../BlockingViews/BlockingView';
+import * as Illustrations from '../Icon/Illustrations';
+
+const propTypes = {
+    /** Child elements */
+    children: PropTypes.node,
+
+    /** If true, child components are replaced with a blocking "not found" view */
+    shouldShow: PropTypes.bool,
```

Comment:
Maybe `shouldBlockContent` will be more intuitive, WDYT?

Reply:
Yes, I think considering name and the way we handle it sounds good. @hayata-suenaga your view?

Reply:
yes that name sounds good 👍 but why do we need a prop for it? This component already subscribes to the screen size. So the information whether to display the blocking view (this component) is already inside the component?

Reply:
or is this just a test component?

Reply:
I agree, as for now, this is not needed. Until such a use case is needed, I believe we should remove all unnecessary props. IMO, this component should be as simple as possible. In the future, if there is a need to block content in a different scenario than the display size condition, it should then be implemented for specific requirements.